common/src/test/java/com/revenuecat/purchases/common/offlineentitlements/PurchasedProductsFetcherTest.kt

```diff
@@ -310,25 +177,22 @@ class PurchasedProductsFetcherTest {
 
         mockEntitlementMapping(mapOfEntitlements)
 
-        val activePurchase = stubStoreTransactionFromGooglePurchase(
+        val activePurchaseMonthly = stubStoreTransactionFromGooglePurchase(
             productIds = listOf(productIdentifierMonthly),
-            purchaseTime = testDate.time
+            purchaseTime = testDate.time,
+            purchaseToken = "test-token-1"
         )
-        val purchaseRecordMonthly = stubStoreTransactionFromPurchaseHistoryRecord(
-            productIds = listOf(productIdentifierMonthly),
-            purchaseTime = testDate.time
-        )
-        val purchaseRecordAnnual = stubStoreTransactionFromPurchaseHistoryRecord(
+        val activePurchaseAnnual = stubStoreTransactionFromGooglePurchase(
             productIds = listOf(productIdentifierAnnual),
-            purchaseTime = testDate.time
+            purchaseTime = testDate.time,
+            purchaseToken = "test-token-2"
         )
 
-        mockAllPurchases(purchaseRecordMonthly, purchaseRecordAnnual)
-        mockActivePurchases(activePurchase)
```

Comment:
I noticed a couple of these tests were wrong, it was actually only mocking a single active purchase, and multiple non-active. They are fixed on this PR.